oaknational/Oak-Web-Application
# Lesson preview page

### Job Story

**When** I am creating lesson content,
**I want to see it in its new state on a secure page**
**So that I can preview my work**.

### Implementation details

- Pages in pupils/beta sit behind owa.thenationalacademy and so are not visible to the public
- NB. this uses new MVs in DBT which will not be available until released to production
- This is a consciously naive implementation. It is likely that some lessons will not render correctly due to missing data.


## How to test

1. Go to https://deploy-preview-2699--oak-web-application.netlify.thenational.academy/pupils/beta/previews/lessons/understanding-dystopias
2. The lesson should be successfully rendered


### Acceptance criteria

- [x]  canonical page at pupils/beta/previews/lesson
- [x]  shows lessons in new state
